core/arch/arm/include/mm/generic_ram_layout.h

```diff
@@ -164,7 +164,6 @@
 #define TA_RAM_START		ROUNDUP(TZDRAM_BASE + TEE_RAM_VA_SIZE, \
 					CORE_MMU_PGDIR_SIZE)
 #define TA_RAM_SIZE		(ROUNDDOWN(TZDRAM_BASE + (TZDRAM_SIZE - \
-					  TEE_RAM_VA_SIZE - \
```

Comment:
good catch. thanks!

Could you make it this way:
```
#ifdef CFG_WITH_PAGER
#define TEE_RAM_START		TZSRAM_BASE
#define TEE_RAM_PH_SIZE		TZSRAM_SIZE
#define TA_RAM_START		ROUNDUP(TZDRAM_BASE, CORE_MMU_PGDIR_SIZE)
#else
#define TEE_RAM_START		TZDRAM_BASE
#define TEE_RAM_PH_SIZE		TEE_RAM_VA_SIZE
#define TA_RAM_START		ROUNDUP(TZDRAM_BASE + TEE_RAM_VA_SIZE, \
					CORE_MMU_PGDIR_SIZE)
#endif /*CFG_WITH_PAGER*/

#define TA_RAM_SIZE		(ROUNDDOWN(TZDRAM_BASE + (TZDRAM_SIZE - \
					   TEE_SDP_TEST_MEM_SIZE), \
					   CORE_MMU_PGDIR_SIZE) - TA_RAM_START)
```

Reply:
Sure, done.